career/ladder.md

```diff
@@ -126,33 +128,36 @@ Here are some examples of other career ladders:
 - [ ] Drives tasks from client research to client acceptance.
 
 #### Simple
-- [ ] Does not over-optimize. Understands when code is "good-enough".
-- [ ] Understands the tradeoffs between technical, analytical, and product needs and leads to solutions that take all of these needs into account.
+- [ ] Has a long track of delivering value for the client, finding the right balance between maintenance work and delivering features. They are recognized as someone who can solve a bottleneck when needed or reduce technical debt.
+- [ ] Consistently demonstrates an understanding of the tradeoffs between technical, analytical, and product needs and leads to solutions that take all of these needs into account.
 
 #### Test-driven
+- [ ] Performs TDD over Test Last (TLD) nearly all the time.
```

Comment:
I'm not sure if this belongs this far up the ladder. 
Could this be on level 3 instead of `Moves from TFD to TFD+Refactor = TDD` 
https://github.com/abtion/guidelines/pull/51/files#diff-87ed49f2dc81712c4cd2c06d164f6f86dfe449980f817da28c9b559dbf1c9612R102